pkg/controllers/updaterun/execution_test.go

```diff
@@ -26,24 +26,24 @@ import (
 	"github.com/kubefleet-dev/kubefleet/pkg/utils/condition"
 )
 
-func TestIsBindingSyncedWithClusterStatus(t *testing.T) {
+func TestGetBindingDiffFromClusterStatus(t *testing.T) {
```

Comment:
is there a UT for exec function?

Reply:
We only have IT for the exec function. I'll add more for concurrent updateRuns in a separate PR.